app.js

```diff
@@ -50,39 +50,35 @@ document.addEventListener('DOMContentLoaded', function() {
 
   function setGuessCount(newCount) {
     guessCount = newCount;
-    document.getElementById("guesses-remaining").innerText = "Guesses remaining: " + guessCount + ".";
+    document.getElementById("guesses-remaining").innerText = `Guesses remaining: ${guessCount}.`;
   }
 
   function toggleClasses(element) {
-    for (var i = 1; i < arguments.length; i++) {
+    for (let i = 1; i < arguments.length; i++) {
       element.classList.toggle(arguments[i]);
```

Comment:
Using ES2015, can you refactor this to remove a reference to the `arguments` array-like object?